### ISSUE TITLE
Two STF Server Share Devices

### ISSUE THREAD

**Issue by BenZii** [openstf/stf#834]
As the title say, now I have two STF Server on two machines,Also they connect to the RethinDB Cluster(share data),but some trouble with me ,how I can  let two STF Server share the devices?once I have one stf server, I use stf provider xxxx to connect devices, but now two server how to share the devices?
Thx a lots.

**Comment by thinkhy** [comment]
You want to make devices on the two servers be usable on STF web UI or use one provider to connect these devices ?

**Comment by BenZii** [comment]
@thinkhy  thanks for your reply。I want to make devices on one server，and usable on two STF web UI。Can it Work？Such STF1 and STF2 can show all devices(devices from one server)，users also can use devices from STF1 or STF2. STF1 and STF2 share the devices Pool. apology for my poor English

**Comment by luoshangyu** [comment]
why two stf servers but extend triproxys? you are trying to solve the performance issue, right?

**Comment by thinkhy** [comment]
@BenZii yes, it's workable for STF to make devices be used across servers. For example, you can deploy your devices on server A, then connect with these devices thru IP and port on server B.
